src/NzbDrone.Core/Download/Clients/Deluge/Deluge.cs

```diff
@@ -37,7 +37,15 @@ public override void MarkItemAsImported(DownloadClientItem downloadClientItem)
             if (Settings.TvImportedCategory.IsNotNullOrWhiteSpace() &&
                 Settings.TvImportedCategory != Settings.TvCategory)
             {
-                _proxy.SetLabel(downloadClientItem.DownloadId.ToLower(), Settings.TvImportedCategory, Settings);
+                try
+                {
+                    _logger.Debug("Setting post-import label {0} in deluge", Settings.TvImportedCategory);
+                    _proxy.SetLabel(downloadClientItem.DownloadId.ToLower(), Settings.TvImportedCategory, Settings);
+                }
+                catch (Exception e)
+                {
+                    _logger.Warn(e, "Deluge failed to set label {0}", Settings.TvImportedCategory);
```

Comment:
This shows up in System->Events, so better `Failed to set Post-Import label {0} in deluge` so the user knows what it's about. Otherwise it's exactly what I meant.